### PR TITLE
Updates to config and adding types module

### PR DESCRIPTION
**Pull request by drewstone**
## Summary of changes
- Changes the configs to use environment variable for RPCURLs and adds parsing.
- Changes the chain name/node name to chain ID everywhere we use it
- Fixes some comments, adds comments and TODOs.
- Changes function names to be more correct.

-----
### Code Checklist 

- [x] Tested
- [x] Documented

### CODE REVIEW

**Review by salman01zp — APPROVED** [review]
LGTM..!

File: tests/lib/localProtocolSubstrate.ts [hunk 1]
```diff
@@ -104,6 +104,7 @@ export class LocalProtocolSubstrate extends SubstrateNodeBase<TypedEvent> {
       runtime: 'WebbProtocol',
       pallets: enabledPallets,
       suri: opts.suri,
+      chainId: 1080,
```
Comment (drewstone) [inline tests/lib/localProtocolSubstrate.ts:107]: Can we get this from the API?

File: tests/lib/localDkg.ts [hunk 2]
```diff
@@ -131,6 +131,7 @@ export class LocalDkg extends SubstrateNodeBase<TypedEvent> {
       runtime: 'DKG',
       pallets: this.opts.enabledPallets ?? [],
       suri: opts.suri,
+      chainId: 1080
```
Comment (drewstone) [inline tests/lib/localDkg.ts:134]: Can we get this from the API @salman01zp ?

File: tests/test/substrate/anchorTransactionRelayer.test.ts [hunk 3]
```diff
@@ -52,6 +52,8 @@ describe('Substrate Anchor Transaction Relayer', function() {
 
   // Governer key
   const PK1 = u8aToHex(ethers.utils.randomBytes(32));
+  // propotocl susbtrate node chainId
+  const chainId = 1080;
```
Comment (drewstone) [inline tests/test/substrate/anchorTransactionRelayer.test.ts:56]: Can we get this from the API?

**Review by drewstone — COMMENTED** [review]
@salman01zp some last comments.

**Pull request merged**